bin/phpcs-audit.sh

```diff
@@ -24,5 +24,5 @@ echo Running PHPCS reports
 for type in ${types[@]}
 do
   echo Running PHPCS report: ${type}
-  ./vendor/bin/phpcs -v -s -p -d memory_limit="256M" --extensions="php" --report="${type}" --report-file="report-${type}${file_affix}.txt" ${file_path}
+  ./vendor/bin/phpcs -v --report="${type}" --report-file="report-${type}${file_affix}.txt" ${file_path}
```

Comment:
What’s plan to handle p and d?

Reply:
`-d memory_limit="256M"` is replaced by `<ini name="memory_limit" value="256M"/>`

Reply:
Unsure about -p... @GaryJones ?

Reply:
Looks like the `-v` option may overrides `-p`

Reply:
From the third commit message: